commons-web/src/main/java/eu/europeana/api/commons/web/model/vocabulary/UserRoles.java

```diff
@@ -0,0 +1,22 @@
+package eu.europeana.api.commons.web.model.vocabulary;
+
+public enum UserRoles {
+
+	USER(new String[]{Operations.RETRIEVE, Operations.CREATE, Operations.DELETE, Operations.UPDATE}), 
+	ADMIN(new String[]{Operations.RETRIEVE, Operations.CREATE, Operations.DELETE, Operations.UPDATE, Operations.ADMIN_ALL, Operations.ADMIN_UNLOCK, Operations.ADMIN_REINDEX, Operations.USER_CREATE, Operations.USER_DELETE}); 
```

Comment:
The user roles are API specific, they must not be included in the API commons, eventually an interface with the methods getName() and getPermissions() can be included in API commons

Reply:
User Roles from entity-web are used and suggested methods implemented in api commons in interface.

Reply:
we must not use plural in class names, please reactor 
eu.europeana.api.commons.definitions.vocabulary.Roles to eu.europeana.api.commons.definitions.vocabulary.Role